ui/components/ui/card/card.js

```diff
@@ -1,57 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Box from '../box';
+import Box, { MultipleSizes } from '../box';
 import {
   BackgroundColor,
   BorderColor,
+  BorderRadius,
   BorderStyle,
   Size,
 } from '../../../helpers/constants/design-system';
 
 const Card = ({
   border = true,
-  padding = 4,
   backgroundColor = BackgroundColor.backgroundDefault,
   children,
   ...props
-}) => {
-  const defaultBorderProps = {
-    borderColor: border && BorderColor.borderMuted,
-    borderRadius: border && Size.MD,
-    borderStyle: border && BorderStyle.solid,
-  };
-
-  return (
-    <Box
-      {...{
-        padding,
-        backgroundColor,
-        ...defaultBorderProps,
-        ...props,
-      }}
-    >
-      {children}
-    </Box>
-  );
-};
+}) => (
+  <Box
+    borderColor={border && BorderColor.borderMuted}
+    borderRadius={border && BorderRadius.MD}
+    borderStyle={border && BorderStyle.solid}
+    padding={4}
+    {...props}
+  >
+    {children}
+  </Box>
+);
 
 Card.propTypes = {
   /**
    * Whether the Card has a border or not.
    * Defaults to true
    */
   border: PropTypes.bool,
-  /**
-   * Padding of the Card component accepts number or an array of 2 numbers.
-   * Defaults to 4 (16px)
-   */
-  padding: Box.propTypes.padding,
   /**
    * The background color of the card
    * Defaults to Color.backgroundDefault
    */
-  backgroundColor: Box.propTypes.backgroundColor,
+  backgroundColor: PropTypes.oneOf(Object.values(BackgroundColor)),
   /**
    * The Card component accepts all Box component props
    */
```

Comment:
Updating card component to follow normal conventions

### Before


https://user-images.githubusercontent.com/8112138/225188956-23bab4b1-1fdb-404e-8364-3dec3558f656.mov

### After


https://user-images.githubusercontent.com/8112138/225189003-d6b325a0-8a89-4e70-9b99-eae09cab6d52.mov